build/MSBuildToolset/Program.cs

```diff
@@ -1,8 +1,107 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-namespace DoNotUse
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Build.Construction;
+
+namespace MSBuildToolset
 {
-    public class Program
+    public sealed class Program
     {
-        public static int Main(string[] args) => 1;
+        public static int Main(string[] args)
+        {
+            if (args.Length != 3)
+            {
+                Console.WriteLine("Usage: RoslynRestore <solutionFile> <nugetExePath> <dotnetPath>");
```

Comment:
Yeah, I had hoped I could set the output exe name. Still trying to figure out how to do that...